### PR TITLE
#311 fix "workflow not found" error

### LINKED ISSUE #311: "workflow not found" error
I see this [here](https://github.com/objectionary/eo/actions/runs/10531730638/job/29184433426):

```
D: Octokit::Client.check_runs_for_ref("objectionary/eo", "dcf609d2fb1ca2...1e1b5d89b6bbcfe8ea8572f8") in 517ms
D: Octokit::Client.workflow_run_job("objectionary/eo", 29098626367) in 94ms
Octokit::NotFound: GET https://api.github.com/repos/objectionary/eo/actions/jobs/29098626367: 404 - Not Found // See: https://docs.github.com/rest/actions/workflow-jobs#get-a-job-for-a-workflow-run
	/usr/local/bundle/gems/octokit-9.1.0/lib/octokit/response/raise_error.rb:14:in `on_complete'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/middleware.rb:57:in `block in call'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/response.rb:42:in `on_complete'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/middleware.rb:56:in `call'
	/usr/local/bundle/gems/faraday-http-cache-2.5.1/lib/faraday/http_cache.rb:286:in `fetch'
	/usr/local/bundle/gems/faraday-http-cache-2.5.1/lib/faraday/http_cache.rb:190:in `process'
	/usr/local/bundle/gems/faraday-http-cache-2.5.1/lib/faraday/http_cache.rb:141:in `call!'
	/usr/local/bundle/gems/faraday-http-cache-2.5.1/lib/faraday/http_cache.rb:121:in `call'
	/usr/local/bundle/gems/fbe-0.0.58/lib/fbe/middleware/quota.rb:48:in `call'
	/usr/local/bundle/gems/faraday-retry-2.2.1/lib/faraday/retry/middleware.rb:153:in `call'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/rack_builder.rb:152:in `build_response'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/connection.rb:444:in `run_request'
	/usr/local/bundle/gems/faraday-2.10.1/lib/faraday/connection.rb:200:in `get'
	/usr/local/bundle/gems/sawyer-0.9.2/lib/sawyer/agent.rb:99:in `call'
	/usr/local/bundle/gems/octokit-9.1.0/lib/octokit/connection.rb:156:in `request'
	/usr/local/bundle/gems/octokit-9.1.0/lib/octokit/connection.rb:19:in `get'
	/usr/local/bundle/gems/octokit-9.1.0/lib/octokit/client/actions_workflow_jobs.rb:17:in `workflow_run_job'
	/usr/local/bundle/gems/verbose-0.0.2/lib/verbose.rb:48:in `method_missing'
	/usr/local/bundle/gems/decoor-0.0.1/lib/decoor.rb:43:in `method_missing'
	/action/judges/github-events/github-events.rb:131:in `block in fetch_workflows'
	/action/judges/github-events/github-events.rb:128:in `each'
	/action/judges/github-events/github-events.rb:128:in `fetch_workflows'
	/action/judges/github-events/github-events.rb:190:in `fill_up_event'
	/action/judges/github-events/github-events.rb:318:in `block (4 levels) in <top (required)>'
```

Maybe we should somehow interpret this situation and avoid the exception.

### PR DESCRIPTION
**Pull request by Suban05**
require merging this pr https://github.com/zerocracy/fbe/pull/82 before
Fixes #311

### CODE REVIEW

**Comment by Suban05** [comment]
@Yegorov please review

File: test/judges/test-github-events.rb [hunk 1]
```diff
@@ -917,12 +917,22 @@ def test_pull_request_event_with_comments
 
   def test_count_numbers_of_workflow_builds
     fb = Factbase.new
-    load_it('github-events', fb, Judges::Options.new({ 'repositories' => 'foo/foo', 'testing' => true }))
-    f = fb.query('(eq what "pull-was-merged")').each.to_a.first
+    repo = 'zerocracy/baza'
+    load_it('github-events', fb, Judges::Options.new({ 'repositories' => repo, 'testing' => true }))
+    f = fb.query("(and (eq what \"pull-was-merged\") (eq branch \"#{repo}\"))").each.to_a.first
```
Comment (Yegorov) [inline test/judges/test-github-events.rb:922]: @Suban05 here the branch name is exactly the repository name?

File: test/judges/test-github-events.rb [hunk 2]
```diff
@@ -917,12 +917,22 @@ def test_pull_request_event_with_comments
...
     assert_equal(4, f.succeeded_builds)
     assert_equal(2, f.failed_builds)
   end
 
+  def test_count_numbers_of_workflow_builds_only_from_github
+    fb = Factbase.new
+    repo = 'zerocracy/judges-action'
+    load_it('github-events', fb, Judges::Options.new({ 'repositories' => repo, 'testing' => true }))
+    f = fb.query("(and (eq what \"pull-was-merged\") (eq branch \"#{repo}\"))").each.to_a.first
```
Comment (Yegorov) [inline test/judges/test-github-events.rb:931]: @Suban05 here the branch name is exactly the repository name?

**Review by Yegorov — COMMENTED** [review]
@Suban05 looks good, but there are several questions

File: test/judges/test-github-events.rb (hunk 1 above)
Comment (Suban05) [reply test/judges/test-github-events.rb:922]: @Yegorov in the tests yes, but in the real data - no. maybe, filter by `event_id`?
like this:
`fb.query('(and (eq what "pull-was-merged") (eq event_id 42))')`

Comment (Yegorov) [reply test/judges/test-github-events.rb:922]: @Suban05 perhaps yes

**Comment by Suban05** [comment]
@Yegorov fixed

**Review by Yegorov — APPROVED** [review]
@Suban05 perfectly!

**Comment by Suban05** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@Suban05 thanks!

**Comment by 0crat** [comment]
@Suban05 Thank you for your contribution! We appreciate your effort, but there are a few areas for improvement. According to our policy, we're deducting points due to the lack of a code review and comments. Your current reward is +16 points, which aligns with our base contribution reward. Please ensure future submissions include a thorough review process to maximize your points. Your running balance remains at [+581](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work and remember to [prioritize](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) both speed and quality in your future contributions.

**Pull request merged**